Formula/vert.x.rb

```diff
@@ -1,13 +1,13 @@
 class VertX < Formula
   desc "Toolkit for building reactive applications on the JVM"
   homepage "https://vertx.io/"
-  url "https://bintray.com/vertx/downloads/download_file?file_path=vert.x-3.9.4-full.zip"
-  sha256 "ea81f1c64333d2372cc9a628c7a7c3491b4ba5b059558fb84137c5b2bf4ecc9e"
+  url "https://search.maven.org/remotecontent?filepath=io/vertx/vertx-stack-manager/4.0.0/vertx-stack-manager-4.0.0-full.tar.gz"
```

Comment:
```suggestion
  url "https://repo1.maven.org/maven2/io/vertx/vertx-stack-manager/4.0.0/vertx-stack-manager-4.0.0-full.tar.gz"
```

It looks like that URL redirects here (which is also the URL I see linked on the [download page](https://repo1.maven.org/maven2/io/vertx/vertx-stack-manager/4.0.0/)). Is there a reason to use the `search.maven.org` URL instead?

Reply:
Because the audit said I should 😁

Reply:
Seems fair, I guess 😅